play-runtime/src/main/scala/play/grpc/internal/PlayRouter.scala

```diff
@@ -68,7 +68,7 @@ import scala.compat.java8.FutureConverters._
    */
   protected val respond: HttpRequest => Future[HttpResponse]
 
-  private val handler = new AkkaHttpHandler {
+  val handler = new AkkaHttpHandler {
```

Comment:
Hmmm, having a public `val` on class under the `play.grpc.internal` sounds strange.

Maybe add a comment indicating the reason or an `@ApiMayChange` or some kind of disclaimer?

Reply:
Maybe make it protected so it can be accessed from the sub-classes (generated code for `Abstract${service.name}Router` and expose a new `val handler:AkkaHttpHandler` from the generated code?

Reply:
I didn't notice that is internal. 
It turns out that this internal class is then extended by the generated `Abstract{Something}ServiceRouter` that is then extended by the user implementation. 

At the end, users are extending classes implementing internal APIs. Not big deal though as we control the generated classes. 

I think `@ApiMayChange` is a good idea. Maybe the real solution would be to keep it protected and have a `PlayWebHandler` accepting a `PlayRouter`, but that's maybe too much. 

Anyway, I will add `@ApiMayChange` for now.

Reply:
The more I look into this, the less I like the solution. 

Having a `PlayWebHandler` is also not good, because it will require a `PlayRouter` that is an internal API.

Reply:
It doesn't sound right to have a `@ApiMayChange` in a class marked with `@InternalApi`, that's my main concern.